lms/templates/edxnotes/note-item.underscore

```diff
@@ -17,7 +17,12 @@
     <ol class="note-comments" role="region" aria-label="<%- gettext('Note') %>">
         <li class="note-comment">
             <p class="note-comment-title"><%- gettext("You commented...") %></p>
-            <p class="note-comment-p"><%= text %></p>
+            <p class="note-comment-p">
+                <%= interpolate_text(_.escape(text), {
+                    highlighted_notes_start: '<span class="note-highlight">',
```

Comment:
My preference is for option 1 because the JavaScript doesn't have to trust that the server performed the necessary escaping. As @muzaffaryousaf says, the code then performs the escaping as late as possible. It also means that the JS has access to the raw text which might be useful in other situations where escaping isn't necessary (e.g. setting the value through JQuery).

@robrap I'm not sure what you mean by "the tag definition (name + delimiters) is in Python". If I'm reading this correctly, the text is the raw bookmark text as stored in the database, and then everything else lives in the Underscore file.